documentation/tutorial.md

```diff
@@ -68,3 +78,660 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
 Model "admin/tutorial" is empty.
 ```
 
+
+## Deploy Charmed MongoDB
+To deploy Charmed MongoDB, fetch the charm from [Charmhub](https://charmhub.io/mongodb?channel=dpe/edge) and deploy it to your model:
+```shell
+juju deploy mongodb --channel dpe/edge
+```
+
+Juju will now fetch Charmed MongoDB and begin deploying it to the LXD cloud. This process can take several minutes depending on how provisioned your machine is. You can track the progress by running:
+```shell
+watch -c juju status --color
+```
+
+This will show charmed MongoDB along with its current status and other helpful information.
+
+Wait until the application is ready, when it is ready `watch -c juju status --color` will show
+```
+Every 2.0s: juju status --color                                                                                         ip-172-31-11-104: Fri Dec  2 11:24:30 2022
+
+Model     Controller  Cloud/Region         Version  SLA          Timestamp
+tutorial  overlord    localhost/localhost  2.9.37   unsupported  11:24:30Z
+
+App      Version  Status  Scale  Charm    Channel   Rev  Exposed  Message
+mongodb           active      1  mongodb  dpe/edge   96  no
+
+Unit        Workload  Agent  Machine  Public address  Ports      Message
+mongodb/0*  active    idle   0        10.23.62.156    27017/tcp
+
+Machine  State    Address       Inst id        Series  AZ  Message
+0        started  10.23.62.156  juju-d35d30-0  focal       Running
+```
+
+## Access MongoDB
+*Disclaimer: this part of the tutorial accesses MongoDB via the `admin` user. **Do not** directly interface with the admin user in a production environment. In a production environment [always create a separate user](https://www.mongodb.com/docs/manual/tutorial/create-users/) and connect to MongoDB with that user instead. Later in the section covering Relations we will cover how to access MongoDB without the admin user.*
+
+For this part of the Tutorial we will access MongoDB - via the MongoDB shell `mongosh`. You can read more about the MongoDB shell [here](https://www.mongodb.com/docs/mongodb-shell/).
+
+
+### Install the MongoDB shell
+While MongoDB is installed within the LXD containers that the application is hosted on, it is not installed on your machine. This means that you will have to install the MongoDB shell yourself. Install `mongosh` by following the official [instructions on how to install the MongoDB shell](https://www.mongodb.com/docs/mongodb-shell/install/#std-label-mdb-shell-install).
```

Comment:
In general, I think it's a good practice to keep a tutorial as self-contained as possible and curate the journey for the learner. If it's possible, try to include the required steps inside this tutorial rather than asking the user to navigate to other pages to follow those steps elsewhere. If the process then becomes very long and arduous and involving many other websites, it can make it more difficult for the user to keep up with the tutorial and they may just abandon it.

Reply:
While following the instructions in the official `mongosh` install instructions, I got this warning about the command that the tutorial asks you to run:

![image](https://user-images.githubusercontent.com/116022293/208158706-0881397e-d9c1-462e-9e5b-2cebf73d169f.png)

Reply:
Also, the most recent build there is Focal. I assume that (using Jammy) I should use that one, but it's not 100% clear.

Reply:
When I ran "sudo apt-get update", this is what I saw:

![image](https://user-images.githubusercontent.com/116022293/208159612-89306146-1d41-4378-803c-50e939e4af5f.png)

Reply:
The instructions also have this (I'm not sure what this means, or if I need to install one or the other of these - I have assumed neither):

![image](https://user-images.githubusercontent.com/116022293/208159823-338c5616-8660-4659-97c2-44d20c0f6829.png)